printer_mods/bythorsthunder/Stealthburner_Rapido_Uhf/README.md

```diff
@@ -0,0 +1,7 @@
+# Stealthburner Rapido UHF
+
+A mod that stretches the Stealthburner Front and Rapido Toolhead Front for use with a Rapido UHF configuration. It moves the lighting and cooling to Stealthburner stock locations relative to new nozzle location.
```

Comment:
Can you specify that this was designed with SB RC1 just in case something changes? might even be a good idea to provide a hard link to the commit of SB you used.

Reply:
No problem.  Forgive my ignorance but should that be a relative link?  Not sure what you mean by a hard link.

Reply:
I meant link to the particular commit you used (instead of just the latest commit, which is default) so that anyone who clicked the link would be shown the exact files you used. the link will look like https://github.com/VoronDesign/Voron-Stealthburner/tree/66831c984ab8d9dfe8dfd114ca31e2aa4bee8c35

Reply:
Ok I think that's all good now.